sql/static/country_codes.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# Creates static tables for mapping between country codes and country names
+# by loading data from CSV files.
+
+cd "$(dirname "$0")"
+
+# List of ISO 3166 alpha-2 country codes and names generated via:
+# curl -sL https://datahub.io/core/country-list/r/data.csv | sed 's/\(.*\),\([A-Za-z]*\).*/\2,\1/' > country_codes.csv
+# See https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
+
+
+## Load country_codes_v1
+
+COUNTRY_CODES_SCHEMA='
+[
+  {
+    "name": "code",
+    "description": "ISO 3166 alpha-2 country code",
+    "type": "STRING",
+    "mode": "REQUIRED"
+  },
+  {
+    "name": "name",
+    "description": "Official country name per ISO 3166",
+    "type": "STRING",
+    "mode": "REQUIRED"
+  }
+]
+'
+
+bq rm -f moz-fx-data-derived-datasets:static.country_codes_v1
+bq mk \
+   --description="Mapping of ISO 3166 alpha-2 country codes to country names; country codes are unique" \
+   --schema=<(echo "$COUNTRY_CODES_SCHEMA") \
+   -t moz-fx-data-derived-datasets:static.country_codes_v1
+
+bq load \
+   --ignore_unknown_values \
+   --skip_leading_rows=1 \
+   --source_format=CSV \
+   --schema=<(echo "$COUNTRY_CODES_SCHEMA") \
+   moz-fx-data-derived-datasets:static.country_codes_v1 \
+   country_codes.csv
+
+
+## Load country_names_v1
+
+COUNTRY_NAMES_SCHEMA='
+[
+  {
+    "name": "name",
+    "description": "An official or common country name; a single country may appear with multiple names",
+    "type": "STRING",
+    "mode": "REQUIRED"
+  },
+  {
+    "name": "code",
+    "description": "ISO 3166 alpha-2 country code",
+    "type": "STRING",
+    "mode": "REQUIRED"
+  }
+]
+'
+
+bq rm -f moz-fx-data-derived-datasets:static.country_names_v1
+bq mk \
+   --description="Mapping of country names to ISO 3166 alpha-2 country codes; the same code may appear under multiple names" \
+   --schema=<(echo "$COUNTRY_NAMES_SCHEMA") \
+   -t moz-fx-data-derived-datasets:static.country_names_v1
+
+# First load the ISO 3166 names.
+bq load \
+   --ignore_unknown_values \
+   --skip_leading_rows=1 \
+   --source_format=CSV \
+   --schema=<(echo "$COUNTRY_CODES_SCHEMA") \
+   moz-fx-data-derived-datasets:static.country_names_v1 \
+   country_codes.csv
```

Comment:
We should do this as a single operation, so that it is atomic and won't cause interruptions
```suggestion
   <(cat country_codes.csv && sed -E '1d;s/(.*),(.*)/\2,\1/' country_names_alternate.csv)
```
or after switching the columns in `country_names_alternate.csv`
```suggestion
   <(cat country_codes.csv && sed 1d country_names_alternate.csv)
```

Reply:
Turns out `bq load` doesn't like getting a named pipe for the final argument, so had to materialize this to a file.